library/Ivoz/Provider/Application/Service/User/SyncFromCsv.php

```diff
@@ -92,31 +95,45 @@ public function execute(CompanyInterface $company, string $csv)
                     $entities[] = $extension;
                 }
 
+                $ddi = null;
                 $notEmptyDdiArgs = count(array_filter($outboundDdiArgs)) > 0;
                 if ($notEmptyDdiArgs) {
                     $ddi = $this->ddiFactory->fromMassProvisioningCsv(
                         $company,
                         ...$outboundDdiArgs
                     );
 
-                    if ($ddi->isNew()) {
-                        $user->setOutgoingDdi($ddi);
+                    $entities[] = $ddi;
+                }
```

Comment:
```suggestion
$missingDdiArgs = count($outboundDdiArgs) !== 3;
$ddi = $missingDdiArgs
   : null
  ? $this->ddiFactory->fromMassProvisioningCsv(
            $company,
            ...$outboundDdiArgs
     );

$entities[] = $ddi;
```

I would rather apply this pattern than having those initializations to null everywhere.